roboplot/core/camera/remote_camera.py

```diff
@@ -9,6 +9,8 @@
 import time
 import cv2
 import numpy as np
+
+import roboplot.core.config as config
 import roboplot.core.camera.camera_client as camera_client
 
 
```

Comment:
For some reason, GitHub won't let me comment on the line this is relevant to, but...
On line 28, you have missed changing a `+` to an `os.path.join`.